### ISSUE TITLE
Feat: Enhance Accessibility: Add ARIA Attributes to Login and Signup Forms

### ISSUE THREAD

**Issue by Avinash905** [UBA-GCOEN/StichHub#455]
### Is there an existing issue raised for this?

- [X] I have searched the existing issues

### Describe the feature

Proposed change: Adding ARIA (Accessible Rich Internet Applications) attributes to our login and signup forms to improve accessibility.

ARIA attributes are crucial for enhancing web application accessibility, benefiting users with disabilities. Currently, our forms lack these attributes, hindering proper interpretation by assistive technologies.

### Record Options

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'23 contributor
- [X] I want to work on this issue

**Comment by Avinash905** [comment]
Please assign me this issue so that I can work on this issue under **GSSOC'23**.

**Comment by Naresh-chandanbatve** [comment]
go for it @Avinash905
